pywb/static/wombat.js

```diff
@@ -3402,12 +3469,17 @@ var _WBWombat = function($wbwindow, wbinfo) {
 
     //============================================
     function default_proxy_get(obj, prop, ownProps) {
-        if (prop == '__WBProxyRealObj__') {
-            return obj;
-        } else if (prop == 'location') {
-            return obj.WB_wombat_location;
-        } else if (prop == "_WB_wombat_obj_proxy") {
-            return obj._WB_wombat_obj_proxy;
+        switch (prop) {
+            case '__WBProxyRealObj__':
+                return obj;
+            case 'location':
+                return obj.WB_wombat_location;
+            case '_WB_wombat_obj_proxy':
+                return obj._WB_wombat_obj_proxy;
+            case 'setTimeout':
```

Comment:
Don't think this override is needed, as it defaults to `obj[prop]` anyway, right?
It should just return the bound function, which should work as expected.

Reply:
Yes the default return value is `obj[prop]` (bound to correct this if function) but since we added the setTimeout and setInterval overrides which handles binding the this arg. 
This statement acts as an early out

Reply:
Yeah, but do we need the early out though? We're not doing it for other overridden functions on window like postMessage, addEventListener, etc... should we just remove it to be consistent? Just thinking if coming back to this later will think that it is needed for some reason for setTimeout/setInterval specifically.. All the other cases are required.